app/datatables/admin_datatable.rb

```diff
@@ -9,6 +9,11 @@ def view_columns
       active: { source: "User.is_active", searchable: false },
       receive_weekly_report: { source: "User.receive_weekly_report", searchable: false },
       created_at: { source: "User.created_at", searchable: false },
+      current_sign_in_at: { source: "User.current_sign_in_at" },
+      last_sign_in_at: { source: "User.last_sign_in_at" },
```

Comment:
Also add `searchable: false` to both of these `_at` fields and `sign_in_count` (date searching isn't very intuitive with data tables)

Reply:
for those `ip` we want to keep them ` searchable: true `?

Reply:
Yeah I don't see why not, if someone wants to paste in an IP and see all logins from that IP address they can